.github/workflows/linting.yml

```
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04 # UDFs fail with "VM error: Internal error: VM crashed" on ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
```
pinning only necessary for builds that really use UDFs. Linting does not.